src/nat/eval/llm_validator.py

```
@@ -0,0 +1,330 @@
+
+        # Add LLM to builder (handles all LLM types)
+        await builder.add_llm(llm_name, llm_config)
+
+        # Try all frameworks to find one that works with this LLM
+        llm = None
+        for framework in LLMFrameworkEnum:
+            try:
+                llm = await builder.get_llm(llm_name, framework)
+                logger.debug("LLM '%s' successfully loaded with framework '%s'", llm_name, framework.value)
+                break  # Found a working framework
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)